simplfy guidance.md (#1)
CondeNast/cnid-tech-tests: infra-task/guidance.md

# Guidance to the candidate

1. We want to make sure that you are a good fit to the culture and the work ethic of the team and company.
1. If possible please return within 3 working days of receiving. It's absolutely fine if you need more time, just let us know.
1. We do not force a minimum or maximum time limit to complete this though we suggest you spend approximately 1-2 hours. Be prepared to talk about what you would do with more time.
1. We want to see what you can achieve within the time that you have: how you approach the task and how you deal with the complexity of the solution.
1. We know you have good infrastructure and operations background. We want you to demonstrate that experience in the short time you take on this task.

> 1. We'd like to see you understand basic application development and deployment, but the infrastructure is the bit we want you to focus on.

Author: ```suggestion
1. We want to see you understand basic application development and deployment, however it is the infrastructure we want you to focus on.
```

# Guidance to the candidate

1. We want to make sure that you are a good fit to the culture and the work ethic of the team and company.

> 1. Please return within 3 working days of receiving if possible, or let us know if you require additional time. 

Author: ```suggestion
1. If possible please return within 3 working days of receiving. It's absolutely fine if you need more time, just let us know.
```

> 1. This prep work is not time restricted - we do not force a minimum or. maximum time limit to complete it - though it is suggested to be completed in approximately 1-2 hours.

Author: ```suggestion
1. We do not force a minimum or maximum time limit to complete this though we suggest you spend approximately 1-2 hours. Be prepared to talk about what you would do with more time.
```

1. We want to see what you can achieve within the time that you have: how you approach the task and how you deal with the complexity of the solution.
1. We know you have good infrastructure and operations background. We want you to demonstrate that experience in the short time you take on this task.
1. We'd like to see you understand basic application development and deployment, but the infrastructure is the bit we want you to focus on.
1. We want to see how you develop your infrastructure and how you apply good DevOps principles.

> 1. We are a software engineering team and have an expectation of your ability to handle code. We want to see that your workflow is suitable to our team. 

Author: ```suggestion
```

1. We want you to apply the information you have gained from talking to us (and others) to the solution if possible.

> 1. Documentation is important! Our team is a group of skilled Infrastructure Developers who work with skilled Application Developers.

Author: ```suggestion
1. Documentation is important! Our team is a group of skilled Infrastructure Developers who work with skilled Application Developers and we like docs that are easy to follow for both.
```
